### PR TITLE
Liuboff/replacing legacy snack bar

### PR DESCRIPTION
**Pull request by Liuboff**
Replacing legacy snack bar.
Task: https://github.com/ita-social-projects/OoS-Frontend/issues/2505

### CODE REVIEW

File: src/app/shared/components/message-bar/message-bar.component.ts [hunk 1]
```diff
@@ -24,4 +24,11 @@ export class MessageBarComponent implements OnInit {
   public onClose(): void {
     this.snackBar.dismiss();
   }
+
+  public onKeyDown(event: KeyboardEvent): void {
+    if (event.key === 'Enter' || event.key === ' ') {
```
Comment (humanmantis) [inline src/app/shared/components/message-bar/message-bar.component.ts]: use `event.keyCode` together with constants  from `@angular/cdk/keycodes`

**Review by humanmantis — CHANGES_REQUESTED** [review]

File: src/app/shared/components/message-bar/message-bar.component.ts (hunk 1 above)
Comment (Liuboff) [reply src/app/shared/components/message-bar/message-bar.component.ts]: Done:
. . .
if (event.keyCode === ENTER || event.keyCode === SPACE) { 
. . .

File: src/app/shared/components/message-bar/message-bar.component.html [hunk 2]
```diff
@@ -10,7 +10,17 @@
...
+  <div matSnackBarActions>
+    <div
+      matSnackBarAction
+      *ngIf="!data.unclosable"
+      (click)="onClose()"
+      (keydown)="onKeyDown($event)"
+      tabindex="0"
+      data-testid="closing-button"
+      class="action"
+    >
```
Comment (humanmantis) [inline src/app/shared/components/message-bar/message-bar.component.html]: Put the `>`  at the end of the last line instead of being alone on the next line

**Review by humanmantis — APPROVED** [review]
Sync branch with develop before merge

File: src/app/shared/components/message-bar/message-bar.component.html (hunk 2 above)
Comment (Liuboff) [reply src/app/shared/components/message-bar/message-bar.component.html]: Done

**Pull request merged**